api.rb

```diff
@@ -25,20 +25,24 @@ def ensure_key_is_set
 
 get "/employees" do
   json RedactedBreatheClient.employees
+rescue RateLimited
+  halt(420, "Rate limited")
 end
 
 get "/absences" do
   json RedactedBreatheClient.absences(employee_id: params[:employee_id], after: params[:after])
+rescue RateLimited
+  halt(420, "Rate limited")
 end
 
 get "/sicknesses" do
   json RedactedBreatheClient.sicknesses(employee_id: params[:employee_id], after: params[:after])
-end
-
-get "/trainings" do
```

Comment:
Let's put this is in a separate commit as it's unrelated to rate limiting

Reply:
:+1: